inc/Engine/Media/PreloadFonts/Frontend/Subscriber.php

```diff
@@ -46,6 +46,7 @@ public static function get_subscribed_events(): array {
 			'rocket_head_items'                   => [ 'add_preload_fonts_in_head', 30 ],
 			'rocket_enable_rucss_fonts_preload'   => 'disable_rucss_preload_fonts',
 			'rocket_preload_fonts_excluded_fonts' => 'get_exclusions',
+			'rocket_buffer'                       => [ 'maybe_remove_existing_preloaded_fonts', 10 ],
```

Comment:
We set the priority to be 10 by default: https://github.com/wp-media/wp-rocket/blob/0aed8bdcc0e25845aa0a3f45c49d2a1b9e502e9f/inc/classes/event-management/class-event-manager.php#L112
```suggestion
			'rocket_buffer'                       => 'maybe_remove_existing_preloaded_fonts',
```